packages/webapp/src/components/Form/form.module.scss

```diff
@@ -2,4 +2,5 @@
   display: flex;
   flex-direction: column;
   flex: 1;
+  height: 100%;
```

Comment:
@Duncan-Brain I saw in the git history that you were just recently working in the styling of Form & Layout in #3509? I used your new prop `fullWidthContent` to drop the padding on the form (👍) but to put the footer buttons at the bottom of the screen -- which it seems they are intended for, based on the JSX below -- I also needed this property.

```js
<Footer
  style={{
    bottom: 0,
    position: hasWhiteBackground ? 'sticky' : 'relative',
    ...classes.footer,
  }}
  hasWhiteBackground={hasWhiteBackground}
>
```

Just wanted to check that this doesn't break whatever adjusts you were just making. Seems okay on the movement task flow, but I'm not sure what I should be checking for!

Reply:
I added it just for mobile view to make the inventory rows full width. It looks good on your branch I think! 

The only thing height could maybe mess up is the max-height of the inventory flex box. It is possible I used a calc() to keep the page from overflowing.